pages/adhoccharges/ad-hoc-charge-object.json

```diff
@@ -2,6 +2,18 @@
   "id": 32,
   "agent": null,
   "appointment": null,
+  "category": {
+    "branch_tax_setup": null,
+    "charge_via_branch": false,
+    "contractor_tax_setup": null,
+    "contractor_usable": false,
+    "default_charge_amount": null,
+    "default_description": null,
+    "default_pay_amount": null,
+    "dft_net_gross": "gross",
+    "id": 11,
+    "name": "Registration fee"
```

Comment:
Try to use example as much as possible please, strings and integers are easy to do, FK i don't care